Makefile

```diff
@@ -1,25 +1,15 @@
-C:=~/src/github.com/coredns/coredns
-
-all:
-	hugo -d /var/www/coredns.io
-
-.PHONY: clean
-clean:
-	rm -rf /var/www/coredns.io/*
-
-.PHONY: test
```

Comment:
this might be still useful?

Reply:
Happy to keep something along the lines of `make server`. Will push that.

Reply:
Kept it as `make run` see latest commit.